src/js/brutusin-json-forms.js

```diff
@@ -222,10 +223,6 @@ if (typeof brutusin === "undefined") {
                         // #46, problem in IE11. TODO polyfill?
                         input.type = "text";
                     }
-                } else if (s.format === "date") {
-                    input.type = "date";
-                } else if (s.format === "time") {
-                    input.type = "time";
                 } else if (s.format === "email") {

```

Comment:
Deletion of above 4 lines unintended. Will add these in my version and re-submit the pull request.